libethcore/KeyManager.cpp

```diff
@@ -342,6 +342,19 @@ string const& KeyManager::accountName(Address const& _address) const
 	}
 }
 
+void KeyManager::changeName(Address const& _address, std::string const& _name)
+{
+	try
+	{
+		m_keyInfo.at(_address).accountName = _name;
+	}
+	catch (...)
+	{
+		return;
+	}
+	write(m_keysFile);
```

Comment:
I assumed that it will not fail. Like it's done with all the other methods in this file.